src/main/java/com/synopsys/integration/alert/workflow/scheduled/PhoneHomeTask.java

```diff
@@ -182,26 +181,24 @@ private PhoneHomeService createPhoneHomeService(ExecutorService phoneHomeExecuto
         return channelMetadata.toArray(String[]::new);
     }
 
-    private Set<String> retrieveChannelMetadataForJobs(List<ConfigurationJobModel> jobs) {
+    private Set<String> retrieveChannelMetadataForJobs(List<DistributionJobModel> jobs) {
         Map<String, Integer> createdDistributions = new HashMap<>();
         String successKeyPart = "::Successes";
-        for (ConfigurationJobModel job : jobs) {
-            for (ConfigurationModel configuration : job.getCopyOfConfigurations()) {
-                String channelName = configuration.getField(ChannelDistributionUIConfig.KEY_CHANNEL_NAME).flatMap(ConfigurationFieldModel::getFieldValue).orElse("");
-                String providerName = configuration.getField(ChannelDistributionUIConfig.KEY_PROVIDER_NAME).flatMap(ConfigurationFieldModel::getFieldValue).orElse("");
-
-                if (StringUtils.isBlank(channelName) || StringUtils.isBlank(providerName)) {
-                    // We want to specifically get the channel configuration here and the only way to determine that is if it has these fields.
-                    continue;
-                }
+        for (DistributionJobModel job : jobs) {
+            String channelName = job.getChannelDescriptorName();
+            String providerName = new BlackDuckProviderKey().getUniversalKey();
```

Comment:
Since this class is a component, you can autowire `BlackDuckProviderKey`.

Reply:
I went vague an autowired a `ProviderKey` if that's alright.

Reply:
Yeah, that's fine. If we ever do need to refactor to multiple providers, it will be with `ProviderKey`.